docs/src/manifolds/stiefel.md

```diff
@@ -44,4 +44,24 @@ Pages = ["manifolds/StiefelCanonicalMetric.jl"]
 Order = [:function]
 ```
 
+# The submersion or normal metric
```

Comment:
```suggestion
## The submersion or normal metric
```

Similar to the other subsections this should also be a `##` (just the main title is a `#`.)